jenkins.sh

```diff
@@ -52,7 +52,7 @@ if [[ ${GIT_BRANCH} != "origin/master" ]]; then
     app spec lib spec config
 fi
 
-GOVUK_CONTENT_SCHEMAS_PATH=tmp/govuk-content-schemas RAILS_ENV=test bundle exec rake
+GOVUK_CONTENT_SCHEMAS_PATH=tmp/govuk-content-schemas RAILS_ENV=test xvfb-run -a bundle exec rake
```

Comment:
To get the specs to run Iocally (in the dev vm) I also had to use this xvfb-run command too, otherwise it would fail the 3 `js: true` features with:

```
Capybara::Webkit::ConnectionError:
       /usr/lib/rbenv/versions/2.2.4/lib/ruby/gems/2.2.0/gems/capybara-webkit-1.7.1/bin/webkit_server failed to start.
```

Is that expected?